app/javascript/controllers/transition_controller.js

```diff
@@ -35,8 +35,13 @@ export default class extends Controller {
     window.dispatchEvent(new CustomEvent('right-column-changed'))
   }
 
-  toggleClassOnce() {
-    if (this.toggled) return
+  toggleClassOn() {
+    if (this.on) return
+    this.toggleClass()
+  }
+
+  toggleClassOff() {
+    if (!this.on) return
     this.toggleClass()
   }
```

Comment:
This was the new method I needed to add (explained below). I went ahead and renamed the `toggleClassOnce()` so that it would be clear with an on & off name.